<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostGameDialog</class>
 <widget class="QDialog" name="HostGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tournament Title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="titleEdit">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ZePs Tournament</string>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="TypeComboBox">
       <item>
        <property name="text">
         <string>Single elimination</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double elimination</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Round robin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Swiss</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Description :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextBrowser" name="descriptionText">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="hostButton">
     <property name="text">
      <string>Host Tournament</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>67</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
